.travis.yml

```diff
@@ -1,58 +1,51 @@
 language: python
-python: 2.7
 sudo: false
+
 cache:
   pip: true
   directories:
     - $HOME/buildout-cache
+
 before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - mkdir $HOME/.buildout
-  - echo "[buildout]" > $HOME/.buildout/default.cfg
-  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
-  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install --upgrade pip setuptools zc.buildout
-  - if [[ `npm -v` != 3.10.7 ]]; then npm i -g npm@3.10.7; fi
-  - npm install -g grunt-cli
-
-env:
-  matrix:
```

Comment:
@tisto sorry I have worked on several things in the mean time and moved files around, so that those links did not work anymore.

Travis currently shows the best information about status, and reflects that we do test the different package types, (well I have tested with all, and excluded the Plone_fattheme_buildout for now, as those seems o be totally brocken and tests always fail) 

Yes I did hat problems with the execution time, the tests has needed more than 2 hours locally and on CI on the first runs of all tests, we are now down to less then 20 Minutes.

I do share your doubts about using tox on CI, it is an unnecessary layer that could make more confusion than actual improvement. The problem is, that CI tools like Travis-CI did not allow that complex configuration that is possible with tools like buildout or tox. So you need an abstraction layer in between to make complex test setups work. 

My Suggestion would be, that we do a larger discussion at the PloneConf, maybe an open space and discuss how to continue. 
As far as I am now, I would work on documentation of my ideas for better best practices including tox, on a separate repository: https://github.com/plone/plone_best_practices_discussion instead of the suggested way of doing that as a pull request on bobtemplates.plone 

This pull request isn't about using tox or things like that, even if I originally started with this intend. This packages was in my opinion in such a bad shape, that it need a cleanup and more tests, so that additionally pull requests could focus on documentation, features, or refactoring. And that is what this pull is about, I did not touch any functionality, or the skeletons beside formatting for readability and making them testable and fixing test errors. 
And a cleanup consists of removing harmful elements.